src/js/view/view.js

```diff
@@ -879,26 +880,13 @@ define([
         }
 
         function _showDisplay() {
-            var model = _instreamMode ? _instreamModel : _model;
-
-            var hasControls = _model.get('controls');
-            var state = model.get('state');
-            if (_display && hasControls && !_audioMode) {
-                if (!_isIPod || state === states.IDLE || state === states.COMPLETE) {
-                    _display.show();
-                }
-            }
-
             // debug this, find out why
-            if (!(_isMobile && model.fullscreen)) {
-                model.getVideo().setControls(false);
+            if (!(_isMobile && _model.fullscreen)) {
+                _model.getVideo().setControls(false);
             }
         }
 
         function _hideDisplay() {
```

Comment:
Seems like we can get rid of this and a few of the calls to it now that there's nothing left in it